### PR TITLE
enh(release): prepare bam 20.04.4 release notes

### PR DESCRIPTION
**Pull request by cgagnaire**
## Description

Prepare BAM 20.04.4 release notes

## Target serie

- [x] 20.04.x
- [ ] 20.10.x (master)

### CODE REVIEW

**Review by chgautier — APPROVED** [review]

File: fr/releases/centreon-commercial-extensions.md [hunk 1]
```diff
@@ -57,6 +57,23 @@ liens
 
 ## Centreon BAM release notes
 
+### 20.04.4
+
+#### Bugfixes
+
+* Les filtres n'étaient pas conservés lorsque la page de supervision était
+  rafraîchit
```
Comment (tuntoja) [inline fr/releases/centreon-commercial-extensions.md:65]: rafraîchie

File: fr/releases/centreon-commercial-extensions.md [hunk 2]
```diff
@@ -57,6 +57,23 @@ liens
...
+  activités métiers
+
+#### Security fixes
+
+* Mise à jour de la librairie Moment.js pour corriger une vulnérabilité
+  aux attaques ReDOS
+
 ### 20.04.3
 
 * Gestion de la compatibility avec HTTP2
```
Comment (tuntoja) [inline fr/releases/centreon-commercial-extensions.md:79]: compatibilité

**Review by tuntoja — CHANGES_REQUESTED** [review]

File: fr/releases/centreon-commercial-extensions.md (hunk 2 above)
Comment (cgagnaire) [inline fr/releases/centreon-commercial-extensions.md:79]:
```suggestion
* Gestion de la compatibilité avec HTTP2
```

File: fr/releases/centreon-commercial-extensions.md (hunk 1 above)
Comment (cgagnaire) [inline fr/releases/centreon-commercial-extensions.md:65]:
```suggestion
  rafraîchie
```

**Pull request merged**